index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Startup Matchmaker</title>
+    <link href='http://fonts.googleapis.com/css?family=Lato:300,700,300italic,700italic' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+</head>
+<body>
+  <div class="mainContainer">
+    <header>
+      <h2>Startup Matchmaker</h2>
+      <p>
```

Comment:
`<p>` are generally used for blocks of text. Instead look to use either a `div` of better yet an `ul` since you have a list of links. 🔨